fractions/microprofile/microprofile-metrics/src/main/java/org/wildfly/swarm/microprofile/metrics/runtime/MetricRegistryFactory.java

```diff
@@ -16,52 +16,34 @@
  */
 package org.wildfly.swarm.microprofile.metrics.runtime;
 
-import org.eclipse.microprofile.metrics.MetricRegistry;
-
-import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.eclipse.microprofile.metrics.MetricRegistry;
 
 /**
  * @author hrupp
  */
 public class MetricRegistryFactory {
 
-    private static final Map<MetricRegistry.Type, MetricRegistry> registries = new HashMap<>();
+    private static final Map<MetricRegistry.Type, MetricRegistry> REGISTRIES = new ConcurrentHashMap<>();
 
-    private MetricRegistryFactory() { /* Singleton */ }
+    private MetricRegistryFactory() {
+        /* Singleton */ }
```

Comment:
:-)

Reply:
Ah, forgot to remove this comment during refactoring... of course, there is not a single instance at all (and never was).

Reply:
And it's not a factory... so let's rename this class to `MetricRegistries`?

Reply:
Yeah, and we could also register an instance of `MetricRegistryRegistry` to a global registry of registry registries ;-)